### ISSUE TITLE
How to get the logit tensor of generated text?

### ISSUE THREAD

**Issue by randomx207** [microsoft/DeepSpeed-MII#368]
Hi, thank you so much for providing this excellent library.

I can't seem to figure out how to get the logit value of the token at generation time. Actually, I just want to get the logit value of the first token generated for each request (i.e. max_new_tokens=1), Instead of getting sampled tokens. How do you do that?

**Comment by mrwyattii** [comment]
Hi @randomx207, this is not something we expose in MII currently. It is on our roadmap to provide additional features like this. I can provide a PR sometime next week to add this!

**Comment by randomx207** [comment]
Thank you for the update and for considering this feature in the roadmap. I appreciate your willingness to provide a PR for it. Looking forward to seeing the enhancements you have in store next week!

**Comment by x66ccff** [comment]
any update on this?

**Comment by Abigail61** [comment]
I encountered the same problem. Has it been resolved?

**Comment by Dragon-Wu** [comment]
one year after, it seems that it still can not output the logits?

**Comment by hyukkyukang** [comment]
Any update on this?
